requirements/tox.txt

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
Change the circleci/config.yml to use install this file instead of `pip install tox`

Reply:
Changed exactly like so.